vrecord

```diff
@@ -33,14 +28,19 @@ BRNG_OUTLIER_THRSHLD=14
 FFMPEG_DECKLINK=(ffmpeg-dl)
 FFPLAY_DECKLINK=(ffplay-dl)
 
+if "${FFMPEG_DECKLINK[@]}" -version 2>&1 | grep "Library not loaded" >/dev/null ; then
+    echo "Please reinstall 'ffmpeg-dl'. brew install amiaopensource/amiaos/ffmpegdecklink Exiting."
```

Comment:
I would suggest something like:
```
    echo "Please reinstall 'ffmpeg-dl':"
    echo "    brew install amiaopensource/amiaos/ffmpegdecklink"
    echo "Exiting."
```
I guess, it’s more understandable. (Or with `cat`.)